Documentation/project-docs/dogfooding.md

```diff
@@ -96,7 +96,8 @@ runtime.
   <PropertyGroup>
```

Comment:
You can delete the "Remember the version number you picked, you'll need it below" line above. It is clearly not helpful and redundant with the comment you have added.

Reply:
Also, I would change the step 1. from "Also, install the specific ..." to "Optionally, install the specific ..."; and renumber the steps to start from 1 instead of 0.

Reply:
done thank's!